src/single-spa-angularjs.js

```diff
@@ -103,22 +101,56 @@ function unmount(opts, mountedInstances, props = {}) {
   });
 }
 
-function getContainerEl(opts, props) {
-  let element;
-  if (opts.domElementGetter) {
-    element = opts.domElementGetter(props);
+function chooseDomElementGetter(opts, props) {
+  if (props.domElement) {
+    return () => props.domElement;
+  } else if (props.domElementGetter) {
+    return props.domElementGetter;
+  } else if (opts.domElementGetter) {
+    return opts.domElementGetter;
   } else {
-    const htmlId = `single-spa-application:${props.name || props.appName}`;
-    element = document.getElementById(htmlId);
-    if (!element) {
-      element = document.createElement("div");
-      element.id = htmlId;
-      document.body.appendChild(element);
+    return defaultDomElementGetter(props);
+  }
+}
+
+function defaultDomElementGetter(props) {
+  const appName = props.appName || props.name;
```

Comment:
I am interested in your thoughts around a breaking change here 😱 

The original [AngularJS version](https://github.com/single-spa/single-spa-angularjs/blob/1631f72268f5f3b5114ece895e82248c9b4b287a/src/single-spa-angularjs.js#L111) has the "or" conditions the other way around 🔄  to the [React convention](https://github.com/single-spa/single-spa-react/blob/044fe8a5fd22cba5c6aaaaa197fac65fd65604bd/src/single-spa-react.js#L199) 🤔 

We could swap them back to stay inline with the AngularJS variant?

Reply:
`props.appName` existed in single-spa@3, but was removed in single-spa@4 and renamed to `name` as part of introducing parcels. Single-spa never passes both `appName` and `name` props, but it's possible for the user to define either of those in its custom props.

I think the risk of your change is here is very unlikely to break things for single-spa-angularjs users, since it's unlikely people are providing an `appName` prop as a custom prop. However, I'm happy to publish this PR as a new major version anyway just to strictly follow semver.